tests/tests/test_filetransfer_cli.py

```diff
@@ -0,0 +1,188 @@
+# Copyright 2021 Northern.tech AS
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+import os
+import subprocess
+import time
+
+from tempfile import NamedTemporaryFile
+
+from ..common_setup import standard_setup_one_client
+from ..MenderAPI import authentication, devauth, get_container_manager, logger
+from .common_connect import wait_for_connect
+from .common import md5sum
+from .mendertesting import MenderTesting
+
+
+class TestFileTransferCLI(MenderTesting):
+    """Tests the port forward functionality"""
```

Comment:
Nit:
`Tests the file transfer functionality`

Reply:
https://github.com/mendersoftware/integration/pull/1274/commits/e5ec5bb00124b8a5dd5abd5a0ccb23910f9e77fc